src/format/opamFile.ml

```diff
@@ -823,10 +826,13 @@ module ConfigSyntax = struct
                (kind,criterion)::List.remove_assoc kind t.solver_criteria }
   let with_solver t solver = { t with solver = Some solver }
 
+  let backup_url_ocamlpro = "http://opam.ocamlpro.com/backup"
```

Comment:
It's not a bug, it's syntactic sugar, which is documented at http://opam.ocaml.org/doc/Manual.html#Generalsyntax.
Empty and absent list fields can be distinguished when needed by using an option rather than a list as the high-level type of the field.

I am not sure this was already the case in `1.2.2`, but recent versions of opam ignore new fields (unless `--strict` was passed) to allow the evolution of the formats. Thus, just adding a field that is advisory doesn't require a new version. But it's true that per-file versions would be more convenient in an agile setting.